src/Components/SearchForm/SearchForm.js

```diff
@@ -82,7 +83,7 @@ const SearchForm = (props) => {
           <option value="faux locs">faux locs</option>
         </select>
         <input
-          className="location-input"
+          className="location-input search-form-input"
```

Comment:
I created a new class so I could add a default margin to each part of the form in the mobile views.